rcqt.so/include/debPackageInfo.h

```diff
@@ -11,6 +11,13 @@ class DebPackageInfo : public PackageInfo{
           cmd[0],
           cmd[1],
           cmd[2]){}
+
+    DebPackageInfo(const std::string& pkgmgr,
+        const std::vector <std::string> cmd):PackageInfo(pkgmgr,
+          cmd[0],
```

Comment:
this again somehow binds to the number. Like a vector size being 3 or more? Why not pass the vector down the hierarchy to base class ctor also and let it unpack it? that would be more extensible and you limit the number of parameters to ctor too.